interface_frontend.py

```diff
@@ -403,7 +403,7 @@ def draw_robot(i, robot, game_state):
     # Robot´lives
     life_label = get_label(
         str(robot.lives),
-        x=92 + 100 * i,
+        x=92 + 98 * i,
```

Comment:
Kdyz se na to cislo divam, prijde mi, ze by bylo mnohem jednodussi pro pristi upravy si ho ulozit do promenne. Takto menis na kazdem radku xkrat stejny udaj.

Reply:
dobrý nápad :-D